### ISSUE TITLE
alice-carousel in css-gird

### ISSUE THREAD

**Issue by b-asaf** [maxmarinich/react-alice-carousel#119]
I am having problems integrating this package in my project.
I am using CSS-GRID for my main layout and when I am trying to integrate the package I have the following behavior: the items appear to be ordered from top to bottom and only when I cancel `transform: translate3d(-916.512px, 0px, 0px);` style.

The first step was to take Alice component [code](https://codesandbox.io/s/pyly505oqm?file=/src/index.js:214-458) and put it inside my code.
So basically, the only change is the wrapper component which is a css-grid
![alice](https://user-images.githubusercontent.com/28925534/81176908-88c64c80-8fae-11ea-854b-ad3e68a85859.PNG)

**Comment by maxmarinich** [comment]
Hi, @b-asaf! Please, provide a code sample that includes your problem.

**Comment by b-asaf** [comment]
I forgot to import the style `'react-alice-carousel/lib/alice-carousel.css';`
my bad, sorry
